### PR TITLE
upd(obs-backgroundremoval): 1.1.3 -> 1.1.6

### PR DESCRIPTION
**Pull request by umireon**

### CODE REVIEW

**Comment by umireon** [comment]
All the systems are included in the incompatible field. I didn't change the incompatible field and I don't know why every system is incompatible.

**Comment by Elsie19** [comment]
Have you tested on supposed incompatible systems?

**Comment by umireon** [comment]
I think we need to add PPA OBS distribution to pacstall repository first.

**Comment by Zahrun** [comment]
> I think we need to add PPA OBS distribution to pacstall repository first.

I met some build dependencies missing in the distros which are in the incompatible field.
One option is to add the ppa to the pacscript, but I preferred to avoid that, so I just excluded the distro versions which couldn’t build with native repos.
I’m fine if you add the ppa though. I think most people will use the -deb version anyway

**Comment by Zahrun** [comment]
> Have you tested on supposed incompatible systems?

I will test these systems again

**Comment by Zahrun** [comment]
Version 1.1.6 builds fine in jammy and is even compatible with obs 27, which was unexpected
image

**Comment by Zahrun** [comment]
Focal fails with "CMake Error: The source directory "/tmp/pacstall/obs-backgroundremoval-1.1.6/linux-ci-x86_64" does not exist.
Specify --help for usage, or press the help button on the CMake GUI.
Error: /tmp/pacstall/obs-backgroundremoval-1.1.6/build_x86_64 is not a directory"
It has cmake 3.16.3 which is supposed to be compatible since https://github.com/royshil/obs-backgroundremoval/pull/250/files

File: packages/obs-backgroundremoval/obs-backgroundremoval.pacscript [hunk 1]
```diff
@@ -1,28 +1,24 @@
...
 pacdeps=("cmake-deb")
 breaks=("${name}-git")
 replace=("${name}")
 arch=('amd64')
 pkgdesc="OBS plugin to replace the background in portrait images and video"
-hash="a3bcca2ad66f86d4d0b793895f517b1b5ffc7046655ca17baf3859d7a79fc75c"
+hash="9c6c0e37c0b938d1ab7112c073ab97d61fa393a79d3bd4d77fb8b61387fa2fbe"
 maintainer="Zahrun <Zahrun@github.com>"
 
 incompatible=('*:jammy' '*:focal' '*:bionic' '*:buster' '*:bullseye')
```
Comment (Zahrun) [inline packages/obs-backgroundremoval/obs-backgroundremoval.pacscript:14]:
```suggestion
incompatible=('*:focal' '*:bionic' '*:buster' '*:bullseye')
```

**Review by Zahrun — CHANGES_REQUESTED** [review]

File: packages/obs-backgroundremoval/obs-backgroundremoval.pacscript [hunk 2]
```diff
@@ -1,28 +1,24 @@
...
 
 build() {
-  mkdir build_x86_64 && cd build_x86_64 || return 1
-  cmake --preset linux-x86_64 -DCMAKE_INSTALL_PREFIX:PATH="${pkgdir}"/usr .. > /dev/null
-  cmake --build . -j"${NCPU}" > /dev/null
+  cmake --preset linux-ci-x86_64 -DCMAKE_INSTALL_PREFIX=/usr
```
[the remark this answers is not in the document]
Comment (umireon) [reply packages/obs-backgroundremoval/obs-backgroundremoval.pacscript:17]: I'm OK with linux-x86_64

**Comment by umireon** [comment]
I don't think our plugin can be built with CMake 3.16.

**Comment by Zahrun** [comment]
> I don't think our plugin can be built with CMake 3.16.

Should line 1 of CMakeList be updated? https://github.com/royshil/obs-backgroundremoval/blob/main/CMakeLists.txt#L1

**Review by Zahrun — APPROVED** [review]
Merge in from master for the checks not to fail

**Comment by Elsie19** [comment]
Ok it works, but do we want to try getting it working on all these distros or no?

**Comment by umireon** [comment]
I don't know why the uninstalling test fails. Does this test need to be passed?

**Comment by Zahrun** [comment]
> Ok it works, but do we want to try getting it working on all these distros or no?

It works on Ubuntu LTS and more recent, Debian stable, testing, and sid. I consider it’s good enough.

> I don't know why the uninstalling test fails. Does this test need to be passed?

This is only a CI issue due to the cmake-deb pacdep. We should be good to merge.

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**